Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1256,11 +1256,15 @@ private final class TokenStreamCreator: SyntaxVisitor {
   }
 
   func visit(_ node: TernaryExprSyntax) -> SyntaxVisitorContinueKind {
-    before(node.questionMark, tokens: .break, .open)
+    before(node.questionMark, tokens: .break(.open(kind: .continuation)), .open)
     after(node.questionMark, tokens: .space)
-    before(node.colonMark, tokens: .break, .open)
+    before(
+      node.colonMark,
+      tokens: .break(.close(mustBreak: false), size: 0), .break(.open(kind: .continuation)), .open)
```

Comment:
That's right—the goal is to prioritize keeping `? foo : bar` together if the ternary wraps. If we close the group before the colon, we could end up with

```swift
condition ? foo
  : bar
```

which would be worse.

Reply:
It might help to put in a brief comment about it. We usually pair the breaks and group tokens. Then again I figured it out pretty quickly. It's up to you.

Reply:
Good idea, done.